javascript/javascript2/week1/js/danishLetters.js

```diff
@@ -0,0 +1,36 @@
+console.log("Find and count the Danish letters\n\n")
+
+function findDanishLetters(array1, array2, object) {
+    for (let index = 0; index < array2.length; index++) {
+        let key = array2[index]
+        if (array1.includes(key)) {
+            object[key]++;
+        }
+    }
+}
+
+function conditionalDelete(object) {
+    for (const key in object) {
+        (object[key] === 0 && key !== 'total') 
+            ? delete object[key]
+            : object['total'] += object[key]
+    }
+}
+
+function danishAlphabets(string) {
+    const letters = { total: 0, æ: 0, ø: 0, å: 0 }
+    const danishLetters = ['æ', 'ø', 'å']
+    const splitString = string.split("");
+
+    findDanishLetters(danishLetters, splitString, letters);
+
+    conditionalDelete(letters);
+
+    return letters;
+}
```

Comment:
Practice with arrow functions which you are going to see and use a lot in future

Reply:
Hi Diny

I agree with you. I thought of using the arrow functions once we finish it in the lesson plan. I started using them in the homework for Javascript-2/week-2 after the class.